__tests__/run_seed.test.js

```diff
@@ -82,6 +82,92 @@ describe("App testing", () => {
         });
     });
   });
+  describe("GET: queries", () => {
+    test("Returns a 200 status and accepts a sort-by query, which sorts articles by any valid column, sorting by created_at by default", () => {
+      return request(app)
+        .get(`/api/articles?sort_by=title`)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles[0].author).toBe("rogersop");
```

Comment:
Please take a look at [jest-sorted](https://www.npmjs.com/package/jest-sorted) for a nice way to check whether this is sorted as expected.

Reply:
At the moment, this is sorted by title, but you're checking the author names?